tools/pymanifold.cpp

```diff
@@ -0,0 +1,169 @@
+#include "manifold.h"
+#include "meshIO.h"
+#include "pybind11/functional.h"
+#include "pybind11/numpy.h"
+#include "pybind11/operators.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+namespace py = pybind11;
+
+using namespace manifold;
+
+void exportManifold(Manifold &m, std::string name) {
+  Mesh out = m.GetMesh();
+  manifold::ExportOptions options;
+  options.faceted = true;
+  options.mat.roughness = 0.2;
+  options.mat.metalness = 0.0;
+  manifold::ExportMesh(name, out, options);
```

Comment:
Of course this is only a subset of our functionality (no MeshRelation for textures and such), but it does seem like a good, simple place to start.